examples/versioning.ipynb

```diff
@@ -0,0 +1,667 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "07297a3d-048b-496b-adb0-8fdd67316021",
+   "metadata": {},
+   "source": [
+    "# Hosts: Working with asset Versions\n",
+    "\n",
+    "This notebook shows how to perform common versioning-related tasks using OpenAssetIO, the OpenAssetIO-MediaCreation traits, and a supporting manager.\n",
+    "\n",
+    "It uses a contrived library that contains several versions of the OpenAssetIO logo."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60bd9de5-64c3-4f15-8824-ff1d94a894d7",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a86e3e7c-6593-4bac-8e6b-c5a107dc702c",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "#### Install packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "80285f6a-8134-4055-9430-a48a5f79e7d3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: openassetio in /Users/tomcowland/dev/OpenAssetIO-MediaCreation/.venv311/lib/python3.11/site-packages (1.0.0a14)\n",
+      "Requirement already satisfied: openassetio-manager-bal in /Users/tomcowland/dev/OpenAssetIO-MediaCreation/.venv311/lib/python3.11/site-packages (1.0.0a11)\n",
+      "Requirement already satisfied: openassetio-mediacreation in /Users/tomcowland/dev/OpenAssetIO-MediaCreation/.venv311/lib/python3.11/site-packages (1.0.0a7)\n",
+      "Requirement already satisfied: Pillow in /Users/tomcowland/dev/OpenAssetIO-MediaCreation/.venv311/lib/python3.11/site-packages (10.0.1)\n",
+      "Requirement already satisfied: importlib-metadata>=3.6 in /Users/tomcowland/dev/OpenAssetIO-MediaCreation/.venv311/lib/python3.11/site-packages (from openassetio) (6.8.0)\n",
```

Comment:
[fix] Don't think we want to ship this, in my viewer it displays this text inline immediately upon load, with your paths in it and everything.

![image](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/assets/114349211/41d9ecfe-f855-43a5-a095-994d37545529)

Reply:
Removed pip install in updated version.